wondrous-app/components/Common/NewInviteLinkModal/styles.tsx

```diff
@@ -0,0 +1,374 @@
+  }
+`;
+
+export const InviteThruLinkMenuItem = styled(MenuItem)`
+  && {
+    background: #121212;
+    color: #c4c4c4;
+    margin: 6px;
+    border-radius: 6px;
+    font-size: 13px;
+    border: 1px solid transparent;
+    height: 34px;
+  }
+
+  &&:hover {
+    background: rgba(0, 0, 0, 1);
+    border: 1px solid #7427ff;
+  }
+
+  .MuiListItem-root.Mui-selected,
+  .MuiListItem-root.Mui-selected:hover {
+    background: none;
+  }
+`;
+
+export const InviteThruLinkButton = styled(Button)`
+  && {
+    border: 1px solid #353535;
+    background: #0f0f0f;
+    font-weight: 400;
+    width: 137px;
+    flex-basis: 137px;
+    height: 40px;
+    border-radius: 6px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 12px 0 0;
+    border-radius: 100px;
+
+    :hover {
+      background: #353535;
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      background: linear-gradient(270deg, #ccbbff -5.62%, #7427ff 45.92%, #00baff 103.12%);
+      -webkit-mask: linear-gradient(#fff 0 0) content-box, linear-gradient(#fff 0 0);
+      -webkit-mask-composite: xor;
+      mask-composite: exclude;
+      padding: 1px;
+      border-radius: 180px;
+    }
+  }
+`;
+
+export const InviteThruLinkButtonLabel = styled(Typography)`
+  && {
+    color: white;
+    font-size: 14px;
+    margin-left: 25px;
+    font-weight: 500;
+  }
+`;
+
+export const InviteThruLinkButtonSuccessLabel = styled(InviteThruLinkButtonLabel)`
+  && {
+    background: #ffffff;
+    background: -webkit-linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
+    background: -moz-linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
+    background: linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    margin-left: 11px;
+  }
+`;
+
+export const LinkSwitch = styled(({ ...props }) => (
+  <FormControlLabel {...props} control={<AndroidSwitch />} label={props.label} />
+))`
+  margin-top: 12px;
+
+  .MuiTypography-body1 {
+    color: #c4c4c4;
+    font-size: 14px;
+  }
+`;
+
+export const InviteThruEmailLabel = styled(Typography)`
+  && {
+    color: #ccbbff;
+    font-size: 14px;
+    font-weight: 500;
+    margin-top: 32px;
+  }
+`;
+
+export const InviteThruEmailTextFieldButtonWrapper = styled.div`
+  display: flex;
+  margin-top: 12px;
+  display: flex;
+  gap: 21px;
+  justify-content: space-between;
+`;
+
+export const InviteThruEmailTextFieldSelectWrapper = styled.div``;
+
+export const InviteThruEmailTextField = styled(TextField)`
+  background: #0f0f0f;
+  border-radius: 6px 0 0 6px;
+  height: 40px;
+  width: 337px;
+  border: 1px solid transparent;
+
+  .MuiInputBase-input {
+    color: #c4c4c4;
+    height: 20px;
+    padding: 10px 15px;
+  }
+
+  .MuiOutlinedInput-input {
+    padding: 11px 15px;
+  }
+`;
+
+export const InviteThruEmailButton = styled(ButtonComponent)`
+  && {
```

Comment:
Can you use this common button `components/Button/index.tsx` or create a new one (here `components/Button/index.tsx`)  to prevent duplicated code?